backend/src/app.ts

```diff
@@ -142,6 +145,42 @@ app.post('/recovery-password', async (req: Request, res: Response) => {
     });
 });
 
+app.post('/reset-password', async (req: Request, res: Response) => {
+  const resetPasswordAction = new ResetPasswordAction(
+    new UserRepositoryPG(dataSource),
+    new ResetTokenRepositoryPG(dataSource)
+  );
+  resetPasswordAction
+    .execute(req.body.userUuid, req.body.token, req.body.password)
+    .then(() => {
+      res.status(200);
+      res.send({ message: 'Password reset!' });
+    })
+    .catch((error) => {
+      switch (error.constructor) {
+        case MandatoryFieldEmptyException:
+          res.status(400);
+          res.send({ message: 'Bad request' });
+          break;
+        case UserNotFoundException:
```

Comment:
Maybe we can add a case to validate short password since we are already doing that when users create an account, what do you think?

Reply:
Yes, It's true! I have forgotten, I will go to add. Thanks Oma!